CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-## dbt-teradata 1.1.0a
+## dbt-teradata 1.2.0a
 
 ### Features
+Connection Retries option added.
+Implemented Grants feature.
```

Comment:
Better to include these feature comments once implementation completed.

Reply:
Sure, edited the file.